google/cloud/spanner_v1/transaction.py

```diff
@@ -185,7 +186,15 @@ def _make_params_pb(params, param_types):
         return {}
 
     def execute_update(
-        self, dml, params=None, param_types=None, query_mode=None, query_options=None
+        self,
+        dml,
+        params=None,
+        param_types=None,
+        query_mode=None,
+        query_options=None,
+        *,
+        retry=gapic_v1.method.DEFAULT,
+        timeout=None,
```

Comment:
changed to gapic_v1.method.DEFAULT in test as well. Thanks for pointing out out, I thought since timeout value has to be a float so we are not setting default as object.